test/Microsoft.AspNetCore.Mvc.Core.Test/FileStreamResultTest.cs

```diff
@@ -45,11 +48,127 @@ public void Constructor_SetsContentTypeAndParameters()
             var result = new FileStreamResult(stream, contentType);
 
             // Assert
+            Assert.False(result.EnableRangeProcessing);
             Assert.Equal(stream, result.FileStream);
             MediaTypeAssert.Equal(expectedMediaType, result.ContentType);
         }
 
         [Fact]
+        public void Constructor_SetsRangeProcessingParameters()
+        {
+            // Arrange
+            var stream = Stream.Null;
+            var contentType = "text/plain";
+            var expectedMediaType = contentType;
+            var lastModified = new DateTimeOffset();
+            var entityTag = new EntityTagHeaderValue("\"Etag\"");
+
+            // Act
+            var result = new FileStreamResult(
+                fileStream: stream,
+                contentType: new MediaTypeHeaderValue(contentType),
+                enableRangeProcessing: true,
+                lastModified: lastModified,
+                entityTag: entityTag);
+
+            // Assert
+            Assert.True(result.EnableRangeProcessing);
+            Assert.Equal(lastModified, result.LastModified);
+            Assert.Equal(entityTag, result.EntityTag);
+            MediaTypeAssert.Equal(expectedMediaType, result.ContentType);
+        }
+
+        [Theory]
+        [InlineData(0, 4, "Hello", 5)]
+        [InlineData(6, 10, "World", 5)]
```

Comment:
@Tratcher can answer if it is supported in StaticFiles but I don't believe it is. But I did add the support in this PR here: https://github.com/aspnet/Mvc/blob/jbagga/ByteRange3702/src/Microsoft.AspNetCore.Mvc.Core/Internal/FileResultExecutorBase.cs#L268-L275. Should I undo that?